pkgs/by-name/je/jellyfin-tui/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  mpv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jellyfin-tui";
+  version = "v1.0.5";
```

Do not use `v` here. Use it only in the fetcher.
```suggestion
  version = "1.0.5";
```

---

Removed the v from version and concatenated v with version in the fetcher itself, not sure if that's the proper way to do it though.

---

That is exactly it. Sorry, I should have clarified.